src/main/java/greencity/entity/OpeningHours.java

```diff
@@ -3,18 +3,16 @@
 import com.fasterxml.jackson.annotation.JsonBackReference;
 import java.time.DayOfWeek;
 import java.time.LocalTime;
-import java.util.List;
 import javax.persistence.*;
-import lombok.AllArgsConstructor;
-import lombok.Builder;
-import lombok.Data;
-import lombok.NoArgsConstructor;
+import lombok.*;
 import org.springframework.format.annotation.DateTimeFormat;
 
 @Entity
-@Data
+@Getter
```

Comment:
Why  annotation  Data changed  to Getter and Setter anotations

Reply:
because I had exception:((